docs/advanced-topics/stack-monitoring.asciidoc

```diff
@@ -0,0 +1,150 @@
+:page_id: stack-monitoring
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+
+[id="{p}-{page_id}"]
+= Stack Monitoring
+
+You can enable link:https://www.elastic.co/guide/en/elasticsearch/reference/current/monitor-elasticsearch-cluster.html[Stack Monitoring]
+on your Elasticsearch and Kibana to collect and ship their metrics and logs.
+
+You just have to reference the separate monitoring Elasticsearch cluster in the `spec.monitoring` section.
```

Comment:
```suggestion
You just have to reference a separate monitoring Elasticsearch cluster in the `spec.monitoring` section.
```

Reply:
Can a user configure the same Elasticsearch cluster to ship logs and metrics? This is a pattern we currently support in ECE, just curious, if the cluster has to be a separate cluster?

Reply:
A user cannot configure an Elasticsearch cluster to monitor itself, the monitoring cluster has to be a separate cluster.
This is not yet supported due to a circular dependency issue (https://github.com/elastic/cloud-on-k8s/issues/4627).

Reply:
I'm adding a note on this limitation.